src/broker/Manager.cc

```diff
@@ -167,17 +199,16 @@ void Manager::InitPostScript()
 
 	broker::broker_options options;
 	options.disable_ssl = get_option("Broker::disable_ssl")->AsBool();
-	options.forward = get_option("Broker::forward_messages")->AsBool();
```

Comment:
I would argue for a desired default of "forwarding on" as I think it'd be good to generally decouple reachability from physical connections: Pub/sub "just works" as long as there's *some* path available. Even for a standard cluster, this will be helpful to avoid requiring the full mesh.

That said, I can see doing this in two steps to avoid surprising: 4.0 leaves it off by default, along a note in NEWS that 4.1 will turn it on. So, that would be (2).

On the question whether to expose this option at all: it'll be good to leave it available, even though I hope most people will never touch it once it's on by default.

Re/ naming: not much of an opinion on whether we should introduce a new option `disable_forwarding` vs reusing the existing `forward`.